tests/performance_tests/README.md

```diff
@@ -0,0 +1,616 @@
+        "resultAvgTps": 15343.875,
+        "expectedTxns": 462000,
+        "resultTxns": 462000,
+        "tpsExpectMet": true,
+        "trxExpectMet": true,
+        "basicTestSuccess": true,
+        "logsDir": "performance_test/2022-10-12_16-55-27/testRunLogs/performance_test_basic/2022-10-12_17-11-11"
+    }
+    },
+    "2": {
+    "success": false,
+    "searchTarget": 15800,
+    "searchFloor": 15400,
+    "searchCeiling": 16200,
+    "basicTestResult": {
+        "targetTPS": 15800,
+        "resultAvgTps": 15523.30357142857,
+        "expectedTxns": 474000,
+        "resultTxns": 474000,
+        "tpsExpectMet": false,
+        "trxExpectMet": true,
+        "basicTestSuccess": true,
+        "logsDir": "performance_test/2022-10-12_16-55-27/testRunLogs/performance_test_basic/2022-10-12_17-13-24"
+    }
+    },
+    "3": {
+    "success": false,
+    "searchTarget": 15600,
+    "searchFloor": 15400,
+    "searchCeiling": 15800,
+    "basicTestResult": {
+        "targetTPS": 15600,
+        "resultAvgTps": 15464.589285714286,
+        "expectedTxns": 468000,
+        "resultTxns": 468000,
+        "tpsExpectMet": false,
+        "trxExpectMet": true,
+        "basicTestSuccess": true,
+        "logsDir": "performance_test/2022-10-12_16-55-27/testRunLogs/performance_test_basic/2022-10-12_17-15-38"
+    }
+    }
+},
+"LongRunningMaxTpsReport": {
+    "Analysis": {
+<truncated>
+    },
+    "args": {
+<truncated>
+    },
+<truncated>
+},
+"args": {
+    "killAll": false,
+    "dontKill": false,
+    "keepLogs": false,
+    "dumpErrorDetails": false,
+    "delay": 1,
+    "nodesFile": null,
+    "verbose": false,
+    "_killEosInstances": true,
+    "_killWallet": true,
+    "pnodes": 1,
+    "totalNodes": 0,
+    "topo": "mesh",
+    "extraNodeosArgs": " --http-max-response-time-ms 990000 --disable-subjective-api-billing true ",
+    "useBiosBootFile": false,
+    "genesisPath": "tests/performance_tests/genesis.json",
+    "maximumP2pPerHost": 5000,
+    "maximumClients": 0,
+    "_totalNodes": 2,
+    "testDurationSec": 10,
+    "finalDurationSec": 30,
+    "maxTpsToTest": 50000,
+    "testIterationMinStep": 500,
+    "tpsLimitPerGenerator": 4000,
+    "saveJsonReport": true,
+    "saveTestJsonReports": false,
+    "numAddlBlocksToPrune": 2,
+    "logsDir": "performance_test/2022-10-12_16-55-27"
+},
+"env": {
+    "system": "Linux",
+    "os": "posix",
+    "release": "5.10.102.1-microsoft-standard-WSL2"
+},
+"nodeosVersion": "v3.2.0-dev"
+}
+```
+</details>
+
+
+### Performance Test Basic
+
+The Performance Test Basic generates a report to detail results of test, statistics around metrics of interest, as well as diagnostic information about the test run.  If `performance_test.py` is run with `--save-test-json`, or `performance_test_basic.py` is run with `--save-json`, the report described below will be written to the timestamped directory within the `performance_test_basic` log directory for the test run with the file name `data.json`.
```

Comment:
"detail results of test" -> "details results of the test"

Reply:
I think what we're actually going for is `to detail results of the test`

Reply:
I think I'm going to go with `report that details results`

Reply:
Addressed: [7a11caa](https://github.com/AntelopeIO/leap/pull/352/commits/7a11caa408cfb61f76cfed2821e15f823727a822)